scipy/_lib/_array_api.py

```diff
@@ -571,3 +587,62 @@ def xp_vector_norm(x: Array, /, *,
     else:
         # to maintain backwards compatibility
         return np.linalg.norm(x, ord=ord, axis=axis, keepdims=keepdims)
+
+
+# work around xp.real raising for floats etc
```

Comment:
Doesn't seem needed as far as I can tell (see `xp_imag` below for a longer comment).